content/blog/2020-08-05-cml-self-hosted-runners-on-demand-with-gpus.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CML self-hosted runners on demand with GPUs
+date: 2020-08-05
+description: |
+  A tutorial on how to setup your own self-hosted runners with GPUs using CML cloud-runners in your CML workflow in Github or Gitlab.
+picture: 2020-08-05/header.png
+author: david_g_ortega
+commentsUrl: https://discuss.dvc.org/t/cml-self-hosted-runners-on-demand-with-gpus/??
+tags:
+  - CML
+  - GPUs
+  - Self-hosted runner
```

Comment:
runner -> Runner

Reply:
I think lowercase might be right here- I usually don't capitalize all words of tags, just the first

Reply:
hmm, those that I checked are capitalized ... obviously not a big deal though :)

Reply:
https://github.com/iterative/dvc.org/issues/1657#issuecomment-668793001 - the list of tags